packages/flex-plugin-e2e-tests/src/utils/pages/view/twilio-console.ts

```diff
@@ -46,14 +40,41 @@ export class TwilioConsole extends Base {
     await this.goto({ baseUrl: this._baseUrl, path });
 
     if (firstLoad) {
-      await this.elementVisible(TwilioConsole._loginForm, `Twilio Console's Login form`);
```

Comment:
Should we consider refactoring to create two separate login methods—one for password and another for token-based authentication? This approach would be useful if we need to switch back to password login in the future, especially since the login flow tends to change. It might be wise to keep this code handy for now.  Wdyt ?

Reply:
The UI based login is never going to work actually. It will always present the captcha so the API login is the only way